src/main/java/org/openjdk/jextract/JextractTool.java

```diff
@@ -82,13 +82,22 @@ private JextractTool(Logger logger) {
         this.logger = logger;
     }
 
-    private static Path generateTmpSource(List<Path> headers) {
-        assert headers.size() > 1;
+    private static boolean isSpecialHeaderName(String str) {
+        return str.startsWith("<") && str.endsWith(">");
+    }
+
+    private static Path generateTmpSource(List<String> headers) {
```

Comment:
This creates a temp file. It would be good to avoid this and maybe use clang APIs to parse in-memory files?

Reply:
we're using the same trick for macro re-parser. I'll try to revisit this in a future PR